File: C++/DS1/Lec2/pattern2.cpp
//Right angled triangle of numbers
#include <iostream>
using namespace std;
int main(){
    int row;
    cout << "Enter the number of rows : ";
    cin >> row;
    int n =1;
    for(int i = 0; i < row; i++){
        for(int j = 0; j <= i; j++){
            cout << n << " ";
            n++;
        }
        cout << endl;
    }
    return 0;
}

File: JAVA/01_core_JAVA/Lecture02/pattern2.java
//Code-Right angled triangle for number
package Lecture02;

import java.util.Scanner;

public class pattern2 {

	public static void main(String[] args) {
		Scanner scn = new Scanner(System.in);
		System.out.print("Please Enter a number: ");
		int n = scn.nextInt();
		int num = 1;
		for (int i = 1; i <= n; i++) {
			for (int j = 1; j <= i; j++) {
				System.out.print(num + "  ");
				num++;
			}
			System.out.println();
		}

	}

}
